qiskit_ibm_runtime/options/utils.py

```diff
@@ -32,20 +33,17 @@ def set_default_error_levels(
     Returns:
         options with correct error level defaults.
     """
+    is_simulator = (
+        isinstance(backend, fake_backend.FakeBackendV2) or backend.configuration().simulator
+    )
```

Comment:
There is no need to change optimization_level and resilience_level for fake backends. Fake backends should always have noise model (from the real backend it mimics), so one doesn't have to pass it in. And none supports resilience_level.

Reply:
Changed so we don't apply this to a FakeBackend.